.github/workflows/development.yml

```diff
@@ -40,4 +38,25 @@ jobs:
       - name: Install dependencies
         run: pip install tox
       - name: Run integration tests
-        run: tox -e test-integration -- -m smoke
+        run: python -m tox -e test-integration -- -m smoke
+
+  publish-dev:
+    needs: [unit-tests, integration-tests]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.8"
+      - name: Install pypa/build and pypa/twine
+        run: python3 -m pip install tox build twine --user
+      - name: Build a binary wheel & Inject build props
```

Comment:
let's create an action for standardization and sharing for tox build pathways in https://github.com/neuralmagic/nm-actions/